src/systems/filecoin_mining/storage_mining/storage_miner_actor.go

```diff
@@ -23,6 +23,9 @@ func (a *StorageMinerActorCode_I) State(rt Runtime) (vmr.ActorStateHandle, State
 
 ////////////////////////////////////////////////////////////////////////////////
 
+// TODO: placeholder
+const MAX_PROVE_COMMIT_SECTOR_EPOCH = block.ChainEpoch(3)
```

Comment:
maybe make it clearer this is a placeholder _value_ and that we'll pick it later, someone may think 3 is actually the number.